docs/source/clouds/cluster.rst

```diff
@@ -301,26 +301,3 @@ Set the ``NCCL_DEBUG=INFO`` environment variable to see the ACTUAL error.
 .. code-block:: bash
 
     python NCCL_DEBUG=INFO train.py ...
-
-
-Distributed sampler
--------------------
-
-Normally now you would need to add a
-:class:`~torch.utils.data.distributed.DistributedSampler` to your dataset, however
-Lightning automates this for you. But if you still need to set a sampler set the Trainer flag
```

Comment:
Maybe want to refer to the section replace_sampler_ddp with a short section? Like you did in the multi-gpu page?

Reply:
I chose to remove this entirely because you can probably expect that anybody looking at the multi node section had a look at the multi-gpu section.

Reply:
then let's remove the entire section 4

Reply:
But the NCCL_DEBUG tip is not in multi_gpu.rst. Do you want me to move it?

Reply:
yes but a whole section with Tips and then only the NCCL debug flag is a bit strange.